torch/_export/non_strict_utils.py

```diff
@@ -136,10 +141,7 @@ def make_fake_inputs(
 
     combined_args = _combine_args(nn_module, args, kwargs)
     _check_dynamic_shapes(combined_args, dynamic_shapes)
-    transformed_dynamic_shapes = _transform_shapes_for_default_dynamic(
-        combined_args, dynamic_shapes
-    )
-    constraints = _process_dynamic_shapes(combined_args, transformed_dynamic_shapes)
+    constraints = _process_dynamic_shapes(combined_args, dynamic_shapes)
```

Comment:
Where does AUTO come in?

Reply:
DimHints now get handled in `_process_dynamic_shapes`